### PR TITLE
Added compute_gene_scores nftest and standardized format of module

### PR DESCRIPTION
**Pull request by nschcolnicov**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/nanostring/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/nanostring _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_scores.config [hunk 1]
```diff
@@ -24,7 +24,7 @@ params {
     config_profile_description = 'Minimal test dataset to check pipeline function'
 
     // Input data
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/nanostring/samplesheets/samplesheet_test.csv'
-    gene_score_yaml = 'https://raw.githubusercontent.com/nf-core/test-datasets/nanostring/gene_score_test.yaml'
+    input           = 'https://raw.githubusercontent.com/nf-core/test-datasets/nanostring/samplesheets/samplesheet_test.csv'
```
Comment (atrigila) [inline conf/test_scores.config:27]: You can add the same `params.pipelines_testdata_base_path`  here, like you did in the line after this one.

File: modules/local/compute_gene_scores/environment.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: compute_gene_scores
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
```
Comment (atrigila) [inline modules/local/compute_gene_scores/environment.yml:5]: defaults channel should be removed from the `environment.yml`.

See:
https://github.com/nf-core/modules/issues/5829

File: modules/local/compute_gene_scores/environment.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+name: compute_gene_scores
```
Comment (atrigila) [inline modules/local/compute_gene_scores/environment.yml:1]: name is also removed in the new nf-core `environment.yml` specification convention.

**Review by atrigila — CHANGES_REQUESTED** [review]

File: modules/local/compute_gene_scores/main.nf [hunk 4]
```diff
@@ -1,7 +1,6 @@
 process COMPUTE_GENE_SCORES {
     label 'process_single'
 
-    conda "r-yaml=2.3.7 r-ggplot2=3.4.4 r-dplyr=1.1.4 r-stringr=1.5.0 bioconductor-gsva=1.46.0 bioconductor-singscore=1.18.0 r-factominer=2.8.0 r-tibble=3.2.1 r-matrixstats=1.1.0"
```
[the remark this answers is not in the document]
Comment (nschcolnicov) [reply modules/local/compute_gene_scores/main.nf:4]: I also tested with conda, and it worked for some reason, I'll update this

**Review by atrigila — APPROVED** [review]

**Pull request merged**